package.json

```diff
@@ -12,7 +12,8 @@
     "lint": "lerna run lint",
     "cs-check": "lerna run cs-check",
     "cs-format": "lerna run cs-format",
-    "dist": "lerna run dist"
+    "build": "lerna run build",
+    "start": "lerna run start --stream --parallel"
```

Comment:
I may not be understanding what exactly you want, but this root level `npm run start` might actually be what you're talking about.

If you run `npm run start` on the root level, it should spin up the playground and live watch the `packages/core` files and recompile upon save. Is that what you're kind of looking for?

Reply:
I believe that if you run `npm start` on the root level, it will watch the `packages/core` file and run the playground -- but the playground that's running will use the `react-jsonschema-form` package from npm and not the actual files that are watched/recompiled. Is that right?

Reply:
Ah yeah, let me see if I can get it to work.

Reply:
> I believe that if you run `npm start` on the root level, it will watch the `packages/core` file and run the playground -- but the playground that's running will use the `react-jsonschema-form` package from npm and not the actual files that are watched/recompiled. Is that right?

unfortunately it does not look like you can run npm run start from the root level (no start defined in package.json)

I can confirm that it uses the playground package from npm when run from the packages/core, as I came across this today, would be good to see if there is a solution to this as I desperately need if then else conditionals as per draft 7 of the json schema spec which OI don't believe is currently implented. I think that having this in will be a more standard compliant way of collapsing of fields as per #268, #304, #598, #920 as basically means if condition is met in data, additional schema is used.

I have no weight here but es6 modules I can understand as they are great for tools such as webpack and rollup allthough not sure if anything is gained building commonjs (browserify???) as I believe the dist files are umd which basically allows commonjs (node), amd (browser i.e requirejs) and straight browser to reference this module.

Reply:
actually managed to get the playground working with my changes, basically case doing build in playground lib and then start core and my changes are there!

documentation / scripts could perhaps do with updating here???

Reply:
I think I have a solution, but I need a better understanding of how you release the docs and playground. @epicfaace would you mind giving me a brief overview?

Reply:
I've added it in #1639

Reply:
Thanks for adding the documentation on #1606! Hopefully I can get around to getting this finalized tomorrow.

I know the `npm link` stuff only works locally, but have we considered using `yarn` instead? I know with `lerna` + `yarn` all that automagical linking will happen under the surface. Here's a quick (outdated) article on it: https://medium.com/@jsilvax/a-workflow-guide-for-lerna-with-yarn-workspaces-60f97481149d.

Reply:
`yarn` would definitely be good, though I think we can still stick with npm for now to minimize changes. As per @erunion 's [excellent suggestion](https://github.com/rjsf-team/react-jsonschema-form/issues/1630#issuecomment-597186108) of using `lerna bootstrap` (which does this automatic linking either with `npm` or with `yarn`), I went ahead and built off your PR to make #1642.

Reply:
Great suggestion!